python/py-nbval/Portfile

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        computationalmodelling nbval 0.9.4
+revision            0
+name                py-${github.project}
+categories-append   science
+
+platforms           darwin
+supported_archs     x86_64
```

Comment:
why is this only ```x86_64``` and not just ```noarch``` (same question applies to ```py-onnx```)?

Reply:
Done thanks.